### PR TITLE
Fix style in vcr configurer

### PR DESCRIPTION
**Pull request by panjan**

### CODE REVIEW

File: spec/vcr_configurer.rb [hunk 1]
```diff
@@ -42,7 +42,7 @@ def self.setup
             :record => vcr_record_mode
           }
 
-          %w(request response).each do |part|
+          %w[request response].each do |part|
```
Comment (yenkins) [inline spec/vcr_configurer.rb]: info: Contains iterators nested 2 deep (NestedIterators) (reek)

**Comment by Hahihula** [comment]
ok to merge

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/872/ : SUCCESS in 7s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/1685/ : SUCCESS in 1m 10s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1946/ : SUCCESS in 1m 10s
- https://jenkins-ii.intgdc.com/job/gooddata-ruby/job/gooddata-ruby-build-pipeline/123/ : SUCCESS in 7m 05s

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-merge-compliance-check/144/ : SUCCESS in 12s

**Pull request merged**